examples/hotrod/README.md

```diff
@@ -49,10 +49,11 @@ go run ./main.go all
 docker run \
   --rm \
   --link jaeger \
+  --env JAEGER_AGENT_HOST=jaeger \
+  --env JAEGER_AGENT_PORT=6831 \
   -p8080-8083:8080-8083 \
   jaegertracing/example-hotrod:1.6 \
```

Comment:
1.6 version is old, so it needs to be updated to accept JAEGER_AGENT_XXX env.
Should I remove version tag so that latest is pulled?

Reply:
This has to be updated. latest is fine or 1.8

Reply:
Changed to latest